lib/gotrue.ex

```diff
@@ -120,7 +120,7 @@ defmodule GoTrue do
     middlewares = [
       {Tesla.Middleware.BaseUrl, @base_url},
       Tesla.Middleware.JSON,
-      {Tesla.Middleware.Headers, authorization: "Bearer #{access_token}"}
+      {Tesla.Middleware.Headers, [{:apikey, @access_token}, {:Authorization, access_token}]}
```

Comment:
- I believe HTTP headers should be lowercased
- Naming is inconsistent, `@access_token`, `access_token`. Maybe we should rename `@access_token` to `@api_key`.

Reply:
After the spec HTTP headers are case-insensitive but  `Authorization`  is usually capitalized. But since `Tesla` always uses lowercase in their examples, I will change it.

Reply:
I applied the changes, also for renaming @access_token to @api_key, but only the module variable. Let me know if you also want to change the config name :)

Reply:
Thanks @treebee!

Yes, let's update the config name to make things consistent.

It a breaking change, but I don't think many are depending on this repo yet.

Reply:
👍 *done*